demo-app/src/open-truss/configs/4-trino-demo.yaml

```diff
@@ -1,5 +1,6 @@
 workflow:
   version: 1
+  debug: true
```

Comment:
I was thinking when reading it I'd normally see my LSP docs say "Debug is used to automatically output debug info to the console when enabled" or something like that.  

Do we have a schema defined yet for this? I'm wondering what we're thinking around the idea of documenting this config options.

Reply:
We don't have the schema documented anywhere, but agreed. We'll probably get to the point soon where we'll want docs for this.